apps/web/src/scenes/UserGalleryPage/UserSharedInfo/UserSharedInfoList/SharedFollowersList.tsx

```diff
@@ -93,7 +93,6 @@ export default function SharedFollowersList({ userRef }: Props) {
           </InfiniteLoader>
         )}
       </AutoSizer>
-
       <VStack></VStack>
```

Comment:
@Robinnnnn does this line w VStack tag have any purpose in this component?

Reply:
wuuut. nah toss it!

Reply:
as long as things look the same afterwards lol

Reply:
coool, removed it and looks like things are the same